chalicelib/check_setup.json

```diff
@@ -147,186 +181,218 @@
         "schedule": {
             "monday_checks": {
                 "cgap": {
-                    "kwargs": {"primary": true},
+                    "kwargs": {
+                        "primary": true
+                    },
                     "dependencies": []
                 }
             }
         }
     },
-    "metawfrs_to_run" : {
-        "title": "Metawfrs_to_run",
+    "metawfrs_to_run": {
+        "title": "MetaWorkflowRuns to run",
         "group": "Pipeline checks",
         "schedule": {
             "fifteen_min_checks_3": {
                 "cgap": {
-                    "kwargs": {"primary": true, "queue_action": "prod"},
+                    "kwargs": {
+                        "primary": true,
+                        "queue_action": "prod"
+                    },
                     "dependencies": []
                 }
             }
         }
     },
-    "metawfrs_to_checkstatus" : {
-        "title": "Metawfrs to check status",
+    "metawfrs_to_checkstatus": {
+        "title": "MetaWorkflowRuns to check status",
         "group": "Pipeline checks",
         "schedule": {
             "fifteen_min_checks": {
                 "cgap": {
-                    "kwargs": {"primary": true, "queue_action": "prod"},
+                    "kwargs": {
+                        "primary": true,
+                        "queue_action": "prod"
+                    },
                     "dependencies": []
                 }
             }
         }
     },
-    "spot_failed_metawfrs" : {
-        "title": "Spot-failed Metawfrs",
+    "spot_failed_metawfrs": {
+        "title": "Spot-failed MetaWorkflowRuns",
         "group": "Pipeline checks",
         "schedule": {
             "fifteen_min_checks_2": {
                 "cgap": {
-                    "kwargs": {"primary": true, "queue_action": "prod"},
+                    "kwargs": {
+                        "primary": true,
+                        "queue_action": "prod"
+                    },
                     "dependencies": []
                 }
             }
         }
     },
-    "failed_metawfrs" : {
-        "title": "Failed Metawfrs",
+    "failed_metawfrs": {
+        "title": "Failed MetaWorkflowRuns",
         "group": "Pipeline checks",
         "schedule": {
             "manual_checks": {
                 "cgap": {
-                    "kwargs": {"primary": true},
+                    "kwargs": {
+                        "primary": true
+                    },
                     "dependencies": []
                 }
             }
         }
     },
-    "metawfrs_to_patch_samples": {
-        "title": "Metawfrs to patch samples with processed files",
+    "metawfrs_to_check_linecount": {
+        "title": "MetaWorkflowRuns for line count qc",
         "group": "Pipeline checks",
         "schedule": {
-            "hourly_checks_2": {
+            "hourly_checks": {
```

Comment:
some of the checks being removed were on the `hourly_checks_2` schedule, you might want to check and see whether the `hourly_checks` and `hourly_checks_2` checks should be rebalanced at all.